doc/radosgw/adminops.rst

```diff
@@ -1163,6 +1163,13 @@ Request Parameters
 :Example: True [True]
 :Required: No
 
+``active``
+
+:Description: Activate or deactive a key.
```

Comment:
s/deactive/deactivate/

Also, perhaps below on new line 1179 "Keys associated with this user account" ?